.eslintrc.json

```diff
@@ -1,14 +1,15 @@
 {
-  "parser": "babel-eslint",
+  "parser": "vue-eslint-parser",
   "parserOptions": {
+    "parser": "babel-eslint",
     "ecmaVersion": 2020,
     "sourceType": "module",
     "babelOptions": {
       "configFile": ".babelrc"
     }
   },
   "plugins": ["prettier"],
-  "extends": ["eslint:recommended", "prettier"],
+  "extends": ["plugin:vue/essential", "eslint:recommended", "@vue/prettier"],
```

Comment:
Can you confirm that this order is correct? IIUC the order can matter here.

Reply:
I think so, this order is from the default vue cli project